freightliner_cascadia_2012/drivers.launch

```diff
@@ -72,6 +72,7 @@ If not using simulated drivers they are activated if the respective mock argumen
 
   <!-- Velodyne Lidar Driver Nodes -->
   <!-- Lidar 1 -->
+  <remap from="driver_discovery" to="/hardware_interface/driver_discovery"/>
```

Comment:
Change this 
```
<remap from="driver_discovery" to="/hardware_interface/driver_discovery"/>
```
to
```
<remap from="driver_discovery" to="$(optenv CARMA_INTR_NS)/driver_discovery"/>
```

This will resolve to /hardware_interface/driver_discovery at run time based on carma.env in the carma config files.